scripts/presubmit-tests.sh

```diff
@@ -311,6 +311,7 @@ function main() {
     if (( RUN_BUILD_TESTS || RUN_UNIT_TESTS || RUN_INTEGRATION_TESTS )); then
       abort "--run-test must be used alone"
     fi
+    (( IS_DOCUMENTATION_PR )) && exit ${failed}
```

Comment:
As discussed offline, for clarity, let's add a check for IS_PRESUBMIT (even though IS_DOCUMENTATION_PR is only set if IS_PRESUBMIT is set).

```
    # If this is a presubmit run, but a documentation-only PR, don't run the test
    (( IS_PRESUBMIT && IS_DOCUMENTATION_PR )) && exit 0
```

Reply:
done